docs/guides/server/keycloak-truststore.adoc

```diff
@@ -38,6 +38,8 @@ For HTTPS requests, this option verifies the hostname of the server's certificat
 * `ANY` means that the hostname is not verified.
 * `WILDCARD` allows wildcards in subdomain names, such as *.foo.com.
 * When using `STRICT`, the Common Name (CN) must match the hostname exactly.
++
+Please note that this settings does not apply to LDAP secure connections which requires strict hostname checking.
```

Comment:
In the `file` property documentation, does it make sense to replace `HTTPS` with `TLS`? The trust store is not only used in LDAP, as you are making clear here.

I think the beginning of this guide is already generic. But perhaps you can spot anything.

Reply:
Thank you, fixed